3-Authorization-II/1-call-api/SPA/src/index.js

```diff
@@ -10,14 +10,32 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./styles/index.css";
 
 /**
- * MSAL should be instantiated outside of the component tree to prevent it from being re-instantiated on re-renders. 
+ * MSAL should be instantiated outside of the component tree to prevent it from being re-instantiated on re-renders.
  * For more, visit: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/getting-started.md
  */
 export const msalInstance = new PublicClientApplication(msalConfig);
```

Comment:
Does this need to be exported?

I'm not seeing used outside of this file.

Reply:
I think it's used in the sample.test.js, but let me check

Reply:
seems like it doesn't -removed